theano/gof/utils.py

```diff
@@ -221,7 +221,7 @@ def __call__(cls, *args, **kwargs):
         for arg in args:
             arg = make_hashable(arg)
             key.append(arg)
-        for k, v in kwargs.iteritems():
+        for k, v in kwargs.items():
```

Comment:
Why? I think it is not good for python2/3 support. Check what is done elsewhere to be sure.